// @langchain/langgraph-sdk/dist/react/index.cjs
Object.defineProperty(exports, Symbol.toStringTag, { value: "Module" });
require("../_virtual/_rolldown/runtime.cjs");
const require_messages = require("../ui/messages.cjs");
let _langchain_react = require("@langchain/react");
//#region src/react/index.ts
function useStream(...args) {
	const options = args[0];
	return (0, _langchain_react.useStream)({
		...options,
		toMessage: require_messages.toMessageDict
	});
}
//#endregion
exports.FetchStreamTransport = _langchain_react.FetchStreamTransport;
Object.defineProperty(exports, "SubagentManager", {
	enumerable: true,
	get: function() {
		return _langchain_react.SubagentManager;
	}
});
Object.defineProperty(exports, "calculateDepthFromNamespace", {
	enumerable: true,
	get: function() {
		return _langchain_react.calculateDepthFromNamespace;
	}
});
Object.defineProperty(exports, "extractParentIdFromNamespace", {
	enumerable: true,
	get: function() {
		return _langchain_react.extractParentIdFromNamespace;
	}
});
Object.defineProperty(exports, "extractToolCallIdFromNamespace", {
	enumerable: true,
	get: function() {
		return _langchain_react.extractToolCallIdFromNamespace;
	}
});
Object.defineProperty(exports, "isSubagentNamespace", {
	enumerable: true,
	get: function() {
		return _langchain_react.isSubagentNamespace;
	}
});
exports.useStream = useStream;
